### ISSUE TITLE
Does this work on co-op?

### ISSUE THREAD

**Issue by luckystrike32111** [Gktwo/KUN-mod#77]

**Comment by sindrava** [comment]
yes it work.. I tried it on my friend's world.. but I recommend only use it on your friend's world and not some random people because they might gonna report you
